tgui-next/packages/tgui/interfaces/BlackmarketDownlink.js

```diff
@@ -0,0 +1,131 @@
+import { map } from 'common/collections';
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { Dimmer, Box, Button, Section, Tabs, AnimatedNumber, Table, Grid } from '../components';
+
+export const BlackmarketDownlink = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  const categories = data.categories || [];
+  const deliveryMethods = data.delivery_methods || [];
+  const deliveryMethodDesc = data.delivery_method_description || [];
+  const markets = data.markets || {};
+  const items = data.items || {};
+
+  const shipmentSelector = !!data.buying && (
+    <Dimmer
+      textAlign="center">
+      <Grid mt={20}>
+        {map(deliveryMethod => {
+          const name = deliveryMethod.name;
+          if (!(name === "LTSRBT" && !data.ltsrbt_built)) {
```

Comment:
Try to use an early `return null` instead of negating this expression. I was really confused while reading this bit. If I understand correctly, you do not show the LTSRBT delivery method if it's not built.

Reply:
What I did should be fine.

Reply:
Alright, thanks. 👍